scripts/apps/authoring/authoring/directives/AuthoringContainerDirective.ts

```diff
@@ -32,6 +32,12 @@ export function AuthoringContainerDirective(authoringWorkspace: AuthoringWorkspa
         scope: {},
         require: 'sdAuthoringContainer',
         link: function(scope, elem, attrs, ctrl) {
+            scope.$on('author_approval:updated', (event) => {
```

Comment:
can you add a comment for the record that it's handled through `resource:updated` event in authoring-react and this one is only needed for angular-based authoring? You should also test that it actually works on authoring react too even if it's not used yet.

Reply:
> // Needed for authoring Angular.

"**Only** needed for authoring Angular." would be better phrasing.